tests/Action/SetObjectFieldValueActionTest.php

```diff
@@ -173,53 +173,6 @@ public function testSetObjectFieldValueActionWithDate(): void
         $this->assertSame(Response::HTTP_OK, $response->getStatusCode());
     }
 
-    public function testSetObjectFieldValueActionWithDateTime(): void
-    {
-        $object = new Bafoo();
```

Comment:
This Bafoo class is used on another test or can be removed?

Reply:
Yes. The `Bafoo` class is still used in `SetObjectFieldValueActionTest::testSetObjectFieldValueActionWithDate()`.